### PR TITLE
[9.3.0]: Update in changelog, galaxy and README.md

### PR DESCRIPTION
**Pull request by ABHISHEK-SINHA10**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Docs Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -27,31 +27,63 @@ OpenManage Ansible Modules simplifies and automates provisioning, deployment, an
...
-    - For offline installation on the Ansible control machine, download the required tar archive version of the collection from [Dell OpenManage collection](https://galaxy.ansible.com/dellemc/openmanage) and run the command given below:  
-      ```ansible-galaxy collection install dellemc-openmanage-<version>.tar.gz```
-
 * From [github](https://github.com/dell/dellemc-openmanage-ansible-modules/tree/collections):  
 Install the collection from the github repository using the latest commit on the branch 'collections'  
 ```ansible-galaxy collection install git+https://github.com/dell/dellemc-openmanage-ansible-modules.git,collections```
 
-## About
+* To Upgrade:  
+Update the collection to the latest version available from the [galaxy](https://galaxy.ansible.com/dellemc/openmanage) and [automationhub](https://console.redhat.com/ansible/automation-hub/repo/published/dellemc/openmanage)
```
Comment (rajshekarp87) [inline README.md]:
```suggestion
Update the collection to the latest version available on [galaxy](https://galaxy.ansible.com/dellemc/openmanage) and [automationhub](https://console.redhat.com/ansible/automation-hub/repo/published/dellemc/openmanage)
```

File: README.md [hunk 2]
```diff
@@ -27,31 +27,63 @@ OpenManage Ansible Modules simplifies and automates provisioning, deployment, an
...
+    * Red Hat Enterprise Linux (RHEL) 9.4 and 8.9
     * SUSE Linux Enterprise Server (SLES) 15 SP5 and 15 SP4
-    * Ubuntu 22.04.4 and 22.04.3
+    * Ubuntu 24.04 and 22.04.4
 
 ## Installation
 
-* From [galaxy](https://galaxy.ansible.com/dellemc/openmanage):  
+* From [galaxy](https://galaxy.ansible.com/dellemc/openmanage) or [automationhub](https://console.redhat.com/ansible/automation-hub/repo/published/dellemc/openmanage):  
+Install the latest collection from the ansible galaxy or automation hub
```
Comment (rajshekarp87) [inline README.md]:
```suggestion
Install the latest collection from the Ansible Galaxy or Automation hub 
```

Comment (ABHISHEK-SINHA10) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (ABHISHEK-SINHA10) [reply README.md]: Done

**Review by rajshekarp87 — DISMISSED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

**Review by sachin-apa — DISMISSED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Pull request merged**